src/main/java/RoleyPoley.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+public class RoleyPoley {
+    public static void main(String[] args) {
+        Task[] taskList = new Task[100];
+        greet();
+        echo(taskList);
+    }
+
+    public static void displayList(Task[] taskList) {
+        for ( int i = 1; i < taskList.length; i++) {
+            if (taskList[i] == null) {
+                break;
+            }
+            System.out.println("\t" + i + ".[" + taskList[i].getStatusIcon() + "]" + taskList[i].description);
+        }
+    }
+
+    public static void createLine() {
+        for (int i = 0; i < 20; i++) {
+            System.out.print('-');
+        }
+        System.out.print('\n');
+    }
+
+    public static void greet() {
+        System.out.println("Hello! I'm RoleyPoley \nWhat can I do for you today?");
```

Comment:
Maybe can put "What can I do for you today?" on the next line using the + operator to improve readability.